### PR TITLE
[debezium] Fix JsonConverter Object NPE when program running on the c…

### PR DESCRIPTION
**Pull request by mincwang**
…luster

### CODE REVIEW

**Comment by mincwang** [comment]
# User feeback example:
- runing on the local:
image
- runing on the cluster:
image

**Comment by mincwang** [comment]
[docs] MySQLSource replace to MySqlSource

When the flink cdc version more than 2.0, MySQLSource should replace to MySqlSource.

**Comment by mincwang** [comment]
- Running test case on the cluster :
 About configuring the relevant code for JsonConverterConfig  only Configure once when a container was deploy.
image

File: flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/JsonDebeziumDeserializationSchema.java [hunk 1]
```diff
@@ -37,21 +37,40 @@
 public class JsonDebeziumDeserializationSchema implements DebeziumDeserializationSchema<String> {
 
     private static final long serialVersionUID = 1L;
+
     private static final JsonConverter CONVERTER = new JsonConverter();
```
[the remark this answers is not in the document]
Comment (mincwang) [reply flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/JsonDebeziumDeserializationSchema.java:42]: yes，can you have any good ways to solution this？

[the remark this answers is not in the document]
Comment (leonardBang) [reply flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/JsonDebeziumDeserializationSchema.java:42]: Sorry for the late reply, you can defined `private transient JsonConverter jsonConverter;` and then lazily initialize it
like

```
if (jsonConverter == null) {
    // initialize it
}
```

[the remark this answers is not in the document]
Comment (mincwang) [reply flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/JsonDebeziumDeserializationSchema.java:42]: all right

**Review by leonardBang — APPROVED** [review]
Thanks @mincwang for the contribution, LGTM, could you also check the change works in cluster?

**Comment by mincwang** [comment]
> Thanks @mincwang for the contribution, LGTM, could you also check the change works in cluster?

ok, I will fix it later along with code style

**Pull request merged**